constants/role.js

```diff
@@ -2,11 +2,11 @@ ADMIN = 1;
 MODERATOR = 2;
 REGISTERED_USER = 3;
 GUEST_USER = 4;
+GUEST_USER = "guest_user";
```

Comment:
Constant declared twice? or did i mess up rebase?

Reply:
I'll make the necessary changes no worries